Source/WebCore/html/HTMLElement.cpp

```diff
@@ -877,14 +878,31 @@ auto HTMLElement::computeDirectionalityFromText() const -> TextDirectionWithStro
         }
     }
 
+    if (RefPtr slot = dynamicDowncast<HTMLSlotElement>(this)) {
+        if (auto* assignedNodes = slot->assignedNodes()) {
```

Comment:
What about fallback content?

Reply:
I don't see any mention in https://html.spec.whatwg.org/#auto-directionality

Reply:
oh, I guess fallback content is taken care of by else case here.